<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_StudentDialog</class>
 <widget class="QDialog" name="Add_StudentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Student Information Setup â€” Add Student</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(34, 34, 34);</string>
  </property>
  <widget class="QPushButton" name="Cancel_pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
font: 900 9pt &quot;Segoe UI Black&quot; rgb(255, 201, 8);
color: rgb(255, 170, 32);
}
QPushButton:Hover
{
color: rgb(255, 170, 32);
background-color: rgb(255, 201, 8);
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>410</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
font: 900 9pt &quot;Segoe UI Black&quot; rgb(255, 201, 8);
color: rgb(255, 170, 32);
}
QPushButton:Hover
{
color: rgb(255, 170, 32);
background-color: rgb(255, 201, 8);
}</string>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>461</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;
background: none;
color: rgb(255, 170, 32);
</string>
      </property>
      <property name="text">
       <string>Last Name</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="SurName">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 246);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;
background: none;
color: rgb(255, 170, 32);
</string>
      </property>
      <property name="text">
       <string>First Name</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="FirstName">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 246);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;
background: none;
color: rgb(255, 170, 32);
</string>
      </property>
      <property name="text">
       <string>Middle Name</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="MiddleName">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 246);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;
background: none;
color: rgb(255, 170, 32);
</string>
      </property>
      <property name="text">
       <string>ID Number</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="IDNumber">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 246);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;
background: none;
color: rgb(255, 170, 32);
</string>
      </property>
      <property name="text">
       <string>Year Level</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="YearLevel">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 246);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;
background: none;
color: rgb(255, 170, 32);
</string>
      </property>
      <property name="text">
       <string>Gender</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Gender">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 246);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;
background: none;
color: rgb(255, 170, 32);
</string>
      </property>
      <property name="text">
       <string>Course Code</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="CourseCode">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 246);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
